docs/Getting Started/Example Apps/index.md

```diff
@@ -2,130 +2,110 @@
-
-There are many flavors to choose from and for now we will only be concerned with the debug versions.
+_Hello Sdl Android_ has been built with different build flavors that allow you to quickly connect the app to an emulator or hardware. You can choose your flavor in the **Build Variant** menu. To open the menu, select **Build > Select Build Variant**. A small window will appear on the bottom left of your IDE that allows you to choose a flavor.
 
-Versions Include:
+There are many flavors to choose from but for now we will only be concerned with the debug build variants:
 
-* `multi` - Multiplexing (Bluetooth, USB, TCP (as secondary transport))
+* `multi` - Multiplexing - Bluetooth, USB, TCP (as secondary transport)
 * `multi_high_bandwidth` - Multiplexing for apps that require a high bandwidth transport
-* `tcp` - Transmission Control Protocol - used only for debugging purposes
+* `tcp` - Transmission Control Protocol - Only used for debugging purposes
 
-We will mainly be dealing with `multi` (if using a TDK) or `tcp` (if connecting to SDL Core via a virtual machine or your localhost, or to Manticore)
+You will mainly be dealing with `multi` build variants if connecting to TDK, or `tcp` if connecting to Manticore or another emulator.
 
-## Transports
+## Connecting to an Infotainment System
+### Emulator
+You can use a simulated or a real device to connect the example app to an emulator. To connect the example app to [Manticore](https://smartdevicelink.com/resources/manticore/) or another emulator, make sure you are using `tcpDebug` build flavor. You must update the IP address and port number in the _Hello Sdl Android_ project so it knows where your emulator is running. Please check the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide for more detailed instructions on how to get the emulator's IP address and port number.
 
-### Configure for TCP
-If you aren't using a TDK or head unit, you can connect to SDL core via a virtual machine or to your localhost. To do this we will use the flavor ```tcpDebug```.
+1. In the main Java folder of _Hello Sdl Android_, open up `SdlService.java`.
+1. At the top of this file, locate the variable declaration for `DEV_MACHINE_IP_ADDRESS` and change it to your emulator's IP address. Set the `TCP_PORT` to your emulator's port number.
 
-For TCP to work, you will have to know the IP address of your machine that is running Sdl Core. If you don't know what it is, running ```ifconfig``` in a linux terminal will usually let you see it for the interface you are connected with to your network. We have to modify the IP address in Hello Sdl Android to let it know where your instance of SDL Core is running.
+        private static final String DEV_MACHINE_IP_ADDRESS = "192.168.1.78"; // Update
```

Comment:
Instead of indenting, use a java code block?

Reply:
If you need to indent to keep the 1, 2, 3, maybe change to a paragraph with "first, second, etc."? Or use `1\.` `2\.`?

Reply:
Fixed.